RoaringBitmap/src/main/java/org/roaringbitmap/FastAggregation.java

```diff
@@ -35,7 +35,7 @@ public static RoaringBitmap and(Iterator<? extends RoaringBitmap> bitmaps) {
    * @return aggregated bitmap
    */
   public static RoaringBitmap and(RoaringBitmap... bitmaps) {
-    if (bitmaps.length > 2) {
+    if (bitmaps.length > 10) {
```

Comment:
optionally '10' could be made a variable with a comment indicating that it is based on a heuristic (so we know it is a rule of thumb).

Reply:
The user can already call the underlying methods if they want to and I think the parameter would be confusing

Reply:
@richardstartin This is a fine answer.